bin/check-uptime.rb

```diff
@@ -35,11 +36,18 @@ class CheckUptime < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 180
 
+  option :reverse,
+         short: '-r',
+         long: '--reverse',
+         description: 'Reverse, Warn if uptime is upper SEC',
```

Comment:
Also the description could be a bit better than what you have. How about "This reverses the comparison on uptime to > thresholds (seconds)"